anchore_engine/clients/grype_wrapper.py

```diff
@@ -231,32 +237,111 @@ def update_grype_db(grype_db_archive_local_file_location: str, version_name: str
             write_lock.release()
 
 
-def get_vulnerabilities(grype_sbom: str) -> json:
+def _get_grype_db_dir():
+    global grype_db_dir
+    return grype_db_dir
+
+
+def _get_grype_db_session():
+    global grype_db_session
+    return grype_db_session
+
+
+def get_current_grype_db_metadata() -> json:
     """
-    Use grype to scan the provided sbom for vulnerabilites.
+    Return the json contents of the metadata file for the in-use version of grype db
     """
-    global grype_db_dir
+    # Get the path to the latest grype_db metadata file
+    latest_grype_db_metadata_file = os.path.join(
+        _get_grype_db_dir(), METADATA_FILE_NAME
+    )
+
+    # Ensure the file exists
+    if not os.path.exists(latest_grype_db_metadata_file):
+        # If not, return None
+        return None
+    else:
```

Comment:
I think this is a personal preference; to me it is more readable this way.

Reply:
I'll defer this one to other reviewers, but in general, most (if not all) non-error-producing comments can be labeled as "personal preference". You might be right that this in particular is a preference to me, but this has some core reasons why that is so:

Adding code that might make it obvious _at the expense of_ making it redundant causes others to double check the intent. In this particular case: "why is there an `else` here? isn't there something else that the next block  shouldn't do? is there a return?". I had to pause and check those because it wasn't clear to me. The `else:` here is redundant, and it doesn't make it any more clear.